### PR TITLE
[13.0][IMP]stock_account_product_run_fifo_hook

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
hook on condition to update standard price.

This is necessary when you don't want the standard price on fifo products to be updated all the time when there is a little change in the price

### CODE REVIEW

File: stock_account_product_run_fifo_hook/hooks.py [hunk 1]
```diff
@@ -62,11 +62,12 @@ def _run_fifo_new(self, quantity, company):
 
         # Update the standard price with the price of the last used candidate,
         # if any.
-        if new_standard_price and self.cost_method == "fifo":
+        # START HOOK update standard price
+        if self._price_updeatable(new_standard_price):
```
Comment (JordiMForgeFlow) [inline stock_account_product_run_fifo_hook/hooks.py]:
```suggestion
        if self._price_updateable(new_standard_price):
```

File: stock_account_product_run_fifo_hook/model/product.py [hunk 2]
```diff
@@ -34,3 +34,7 @@ def _get_candidates_domain(self, company):
             ("company_id", "=", company.id),
         ]
         return candidates_domain
+
+    def _price_updeatable(self, new_standard_price=False):
```
Comment (JordiMForgeFlow) [inline stock_account_product_run_fifo_hook/model/product.py]:
```suggestion
    def _price_updateable(self, new_standard_price=False):
```

File: stock_account_product_run_fifo_hook/hooks.py [hunk 3]
```diff
@@ -62,11 +62,12 @@ def _run_fifo_new(self, quantity, company):
 
         # Update the standard price with the price of the last used candidate,
         # if any.
-        if new_standard_price and self.cost_method == "fifo":
+        # START HOOK update standard price
+        if self._price_updateable(new_standard_price):
```
Comment (AdriaGForgeFlow) [inline stock_account_product_run_fifo_hook/hooks.py]: Maybe you should add also something to check this function exists like here: https://github.com/ForgeFlow/stock-logistics-workflow/pull/3/files#diff-eb0783cb5bfd7be2476efed449d4f7284911d91d693449d0410ed93b24206c61R11

Comment (AaronHForgeFlow) [reply stock_account_product_run_fifo_hook/hooks.py]: Sure, can you show me an example of how to do that? I don't remember to do anything similar in the past Thanks in advance

Comment (AaronHForgeFlow) [reply stock_account_product_run_fifo_hook/hooks.py]: I think the example is in the same file xD sorry

Comment (AaronHForgeFlow) [reply stock_account_product_run_fifo_hook/hooks.py]: Does it look correct now?

**Review by JordiMForgeFlow — APPROVED** [review]
Code review LGTM 👍🏼

**Pull request merged**